commands/service_docs.go

```diff
@@ -26,15 +26,17 @@ func newServiceDocsCmd(e ServiceExecutor) *serviceDocsCmd {
 		Use:   "gen-doc",
 		Short: "Generate the documentation for the service in a README.md file",
 		Example: `mesg-core service gen-doc
-mesg-core service gen-doc ./PATH_TO_SERVICE`,
+mesg-core service gen-doc ./PATH_TO_SERVICE
+mesg-core service gen-doc --force`,
```

Comment:
```suggestion
mesg-core service gen-doc --force ./PATH_TO_SERVICE`,
```

Reply:
The path is optional, that's why it is only show once.